webapp/src/views/projects/dashboard/LanguageStats/LanguageLabels.tsx

```diff
@@ -94,7 +82,7 @@ export const LanguageLabels: React.FC<Props> = ({ data }) => {
                 <T keyName={translationStates[item.status].translationKey} />
               </Box>
               <Box ml={2} data-cy="project-dashboard-language-label-percentage">
-                {formatPercentage(item.percent)}
+                {formatPercent(item.percent)}
```

Comment:
This should be a component, when it returns components, shouldn't it?

Reply:
Ok